src/core/geometry/poly3/measureBoundingSphere.js

```diff
@@ -2,21 +2,15 @@ const vec3 = require('../../math/vec3')
 const measureBoundingBox = require('./measureBoundingBox')
 
 /** compute's a poly3's bounding sphere
- * returns an array with a vec3 (center point) and a radius
+ * returns an array with a midpoint (vec3) and a radius
```

Comment:
I do not agree with the naming of this, center makes sense, midpoint does not :)
it really is the center of the bounding sphere, literally
(in local coordinates)